source/docs/advanced/features/flash-messages.md

```diff
@@ -1,11 +1,10 @@
 ---
 id: flash-messages
 title: Flash Messages
+description: Your server actions may need to display information to the User after a redirection. This guide explains how to use Front-Commerce's flash messages to achieve this.
```

Comment:
Are the capitalization on "User" and "Customer" wanted? There seem to be some inconsistencies throughout the documentation

Reply:
Not really. I think we can keep it lowercase.
I'll update everything at once in a separate global dedicated MR.